test/unit/model/consent.test.ts

```diff
@@ -172,7 +193,64 @@ describe('src/model/consent', (): void => {
 
       expect(consents[0].id).toEqual(partialConsent.id)
       expect(consents[0].createdAt).toEqual(expect.any(String))
-      expect(consents[0]).toEqual(expect.objectContaining(completeConsent))
+      expect(consents[0]).toEqual(expect.objectContaining(consentWithOnlyUpdateFields))
+    })
+
+    it('updates existing consent with only non-conflicting fields from a consent', async (): Promise<void> => {
```

Comment:
you should explain here, maybe in comments, or better in code, the meaning of `conflicting fields`